makefile

```diff
@@ -4,9 +4,8 @@ COMPOSE_FILE_STANDARD = docker-compose.yml
 TEST_COMMAND = pytest tests/test_api_endpoints.py
 LOCAL_DATABASE = ./database/docker-compose.yml
 DATABASE_TABLES = users, followers, messages, latest
-DELAY_TEST_EXECUTION_SECONDS = 3
+DELAY_TEST_EXECUTION_SECONDS = 30
```

Comment:
Maybe discard your changes. Think what we had before was enough

Reply:
Roger that. Changed back to 3